editor/src/components/canvas/canvas-strategies/strategies/group-helpers.ts

```diff
@@ -96,38 +107,68 @@ function elementHasPercentagePins(jsxElement: JSXElement): boolean {
 }
 
 function elementHasValidPins(jsxElement: JSXElement): boolean {
-  function isValidPin(name: StyleLayoutProp): boolean {
+  function getPin(name: StyleLayoutProp): CSSNumber | null {
     const pin = getLayoutProperty(name, right(jsxElement.props), styleStringInArray)
-    return isRight(pin) && isCSSNumber(pin.value)
+    return isRight(pin) && isCSSNumber(pin.value) ? pin.value : null
   }
-  const leftPin = isValidPin('left')
-  const rightPin = isValidPin('right')
-  const topPin = isValidPin('top')
-  const bottomPin = isValidPin('bottom')
+  const leftPin = getPin('left')
+  const rightPin = getPin('right')
+  const topPin = getPin('top')
+  const bottomPin = getPin('bottom')
+  const width = getPin('width')
+  const height = getPin('height')
 
-  if (leftPin || rightPin) {
-    return topPin || bottomPin
+  if (leftPin != null || rightPin != null) {
+    return topPin != null || bottomPin != null || height?.unit === '%'
   } else {
-    return !(topPin || bottomPin)
+    return !(topPin != null || bottomPin != null) || width?.unit === '%'
```

Comment:
This is equivalent right?
```suggestion
    return (topPin == null && bottomPin == null) || width?.unit === '%'
```

Reply:
yep but for some reason it was easier on my brain in that form - updating it!